packages/common/src/FormattingUtils.js

```diff
@@ -124,9 +124,9 @@ function addSign(number) {
   }
 }
 
-// Take an amount based on fromDecimals and convert it to an amount based on toDecimals
-// For example amount = 100 usdt, its decimals are 6. You want to convert it to 18.
-// convertDecimals(6,18)(100000000)  => 1000000000000000000000.
+// Take an amount based on fromDecimals and convert it to an amount based on toDecimals. For example 100 usdt = 100e6,
+// with 6 decimals. If you wanted to convert this to a base 18 decimals you would get:
```

Comment:
It looks like this function catches both cases, where `fromDecimals > toDecimals` and `fromDecimals < toDecimals`, right?